lib/endpoints/class-wp-rest-controller.php

```diff
@@ -250,7 +250,6 @@ public function get_collection_params() {
 				'default'            => 10,
 				'minimum'            => 1,
 				'maximum'            => 100,
-				'sanitize_callback'  => 'absint',
```

Comment:
`per_page=-1` will be changed to `per_page=1` by `absint()`.
So API will returns a single post and status header will be `200`.
If this line would be removed, API returns `400`.